org.mixedrealitytoolkit.spatialmanipulation/Solvers/Solver.cs

```diff
@@ -15,12 +17,21 @@ namespace MixedReality.Toolkit.SpatialManipulation
     [HelpURL("https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/features/ux-building-blocks/solvers/solver")]
     public abstract class Solver : MonoBehaviour
     {
-        private static ControllerLookup controllerLookup;
+        [Obsolete]
```

Comment:
Add message parameter

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/749/commits/c16545c686519bafd6fb5ffd75360b780829a71f , resolving.